_config/features_comparison.yml

```diff
@@ -0,0 +1,241 @@
+features:
+  - name: "ACID*"
+    jta: true
+    jts: true
+    xts: true
+    comp: false
+    rts: true
+    stm: true
+    blacktie: true
+    arjuna: true
+    txoj: true
+  - name: "Synchronizations"
+    jta: true
+    jts: true
+    xts: true
+    comp: false
+    rts: true
+    stm: false
+    blacktie: false
+    arjuna: true
+    txoj: false
+  - name: "Nested transactions"
+    jta: true
+    jts: true
+    xts: true
+    comp: false
+    rts: false
+    stm: true
+    blacktie: false
+    arjuna: true
+    txoj: true
+  - name: "Automated failure recovery"
+    jta: true
+    jts: true
+    xts: true
+    comp: false
+    rts: true
+    stm: false
+    blacktie: true
+    arjuna: true
+    txoj: true
+  - name: "Transaction log (file system, JDBC, HornetQ)"
+    jta: true
+    jts: true
+    xts: true
+    comp: false
+    rts: true
+    stm: true
+    blacktie: true
+    arjuna: true
+    txoj: true
+  - name: "JTA 1.2 API"
+    jta: true
+    jts: true
+    xts: true
+    comp: false
+    rts: false
+    stm: false
+    blacktie: false
+    arjuna: false
+    txoj: false
+  - name: "XAResource"
+    jta: true
+    jts: true
+    xts: true
+    comp: false
+    rts: true
+    stm: false
+    blacktie: true
+    arjuna: false
+    txoj: false
+  - name: "JTA bridging"
+    jta: false
+    jts: false
+    xts: true
+    comp: false
+    rts: true
+    stm: false
+    blacktie: false
+    arjuna: false
+    txoj: false
+  - name: "Provides Annotations API"
+    jta: true
+    jts: true
+    xts: true
+    comp: true
+    rts: false
+    stm: true
+    blacktie: false
+    arjuna: false
+    txoj: false
+  - name: "Local transactions"
```

Comment:
So you should rename it to

bq. "Has Protocol Optimizations when all Actors are running in the same VM"

Personally I would say this row does not add any value.

JTS calls to in-VM actors still go over CORBA so JTS also needs to have this check box ticked.